src/login/Login.jsx

```diff
@@ -0,0 +1,133 @@
+
+        };
+        let data = "";
+        let resStatus = 0;
+        fetch("http://127.0.0.1:5000/login", requestLogin)
+            .then(async response => {
+                resStatus = response.status
+                data = await response.json();
+            })
+            .then(res => {
+                switch (resStatus) {
+                    case 200:
+                        setAccessToken(data["access_token"]);
+                        setAccessExpiry(data["access_expiry"]);
+                        setIsAuthenticated(true);
+                        break
+                    default:
+                        setResponseMessage(data["message"]);
+                        break
+                }
+            })
+            .catch(error => {
+                setResponseMessage(data["message"]);
+            });
+    }
+
+    if (isAuthenticated === true) {
+        return <Redirect to="/my_space" />
+    }
+
+    return (
+        <div className="container">
+            <div className="row mb-5">
+                <div className="col-lg-12 text-center">
+                    <h1 className="mt-5">Login</h1>
+                </div>
+            </div>
+            <div className="row">
+                <div className="col-lg-12">
+                    <form className="login-form mx-auto" onSubmit={handleSubmit}>
+                        <form-group controlId="formUserame">
+                            <p className="input-control">
+                                <label htmlFor="username">Username or Email:</label>
+                                <input className="field"
+                                    type="text"
+                                    name="username"
+                                    placeholder="Username or Email"
+                                    onChange={e => setUsername(e.target.value)}
+                                    required
+                                />
+                            </p>
+                        </form-group>
+                        <div><br></br></div>
+                        <form-group controlId="formPassword">
+                            <p className="input-control">
+                                <label htmlFor="password">Password :</label>
+                                <input className="field"
+                                    type="password"
+                                    name="password"
+                                    placeholder="Password"
+                                    onChange={e => setPassword(e.target.value)}
+                                    required
+                                />
+                            </p>
+                        </form-group>
+                        <div><br></br></div>
+                        <div>
+                            {responseMessage !== null && <span className="error" name="response" aria-label="response" role="alert">{responseMessage}</span>}
```

Comment:
since this responseMessage will only show up in case there is an error, can we call this errorMessage?

Reply:
done